Readme.md

```diff
@@ -43,7 +43,7 @@ You can check out more about the project direction in the [vision document](Visi
 
 ## Sample Project
 
-There's a sample project in the Demo directory. To use it, run `pod install` to download the required libraries. Have fun!
+There's a sample project in the Demo directory. To use it, run `carthage update` to download the required libraries. Have fun!
```

Comment:
@BasThomas Looks related to #1499. It's possible we can't run the demo with `pod install` because the Quick/Nimble dependencies are managed with Carthage`?

Reply:
The project don't have `Podfile`, maybe have been switch to `carthage`.

Reply:
Examples were merged with main project so we don’t use CocoaPods anymore.